src/microbe_stage/editor/MicrobeEditor.tscn

```diff
@@ -7483,7 +7483,7 @@ margin_right = 17.0
 margin_bottom = 17.0
 custom_fonts/font = SubResource( 46 )
 custom_colors/font_color = Color( 1, 0.301961, 0.301961, 1 )
-text = "+2"
+text = "+200%"
```

Comment:
Hang on are you changing the osmoregulation shown on organelles? That's completely wrong. That is a flat increase.
The membranes are which add a percentage modifier to the osmoregulation calculation.